src/pages/vendor/VendorProductsPage/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Grid, Box } from '@mui/material';
+
+import { sortOptions } from 'src/common/constants';
+import AdminSectionSubTitle from 'src/pages/admin/AdminSectionSubTitle';
+import SortButton from 'src/pages/admin/SortButton';
+import { SortOrder } from 'src/redux/user/types';
+
+import VendorSectionTitle from '../VendorSectionTitle';
+
+import ProductsTable from './ProductsTable';
+import products from './ProductsTable/productsMock';
+import SectionWrapper from './styles';
+
+function VendorProductPage() {
+  const { t } = useTranslation();
+  const [, setOrder] = useState<SortOrder>(sortOptions.DESC);
```

Comment:
why state needed if value is not used

Reply:
I will use order later when I'll be connecting to backend and send order in request body, and backend will sort products in need order